docs/documentation/platform/pam/getting-started/resources/ssh.mdx

```diff
@@ -0,0 +1,267 @@
+description: "Learn how to configure SSH server access through Infisical PAM with support for password, key-based, and certificate-based authentication."
+---
+
+Infisical PAM supports secure, just-in-time access to SSH servers. This allows your team to access Linux/Unix servers without sharing static credentials, while maintaining a complete audit trail of all sessions.
+
+## How It Works
+
+SSH access in Infisical PAM uses a Gateway deployed in your network to establish secure connections to your servers. The Gateway acts as a bridge between Infisical and your private infrastructure.
+
+```mermaid
+sequenceDiagram
+    participant CLI as Infisical CLI
+    participant Gateway
+    participant Infisical
+    participant SSH as SSH Server
+
+    CLI->>Infisical: Request session (authenticate)
+    Infisical-->>CLI: Session ID + Gateway connection details
+    CLI->>Gateway: Connect via secure tunnel
+    Gateway->>Infisical: Fetch credentials for session
+    Infisical-->>Gateway: Return credentials
+    Gateway->>SSH: Connect with credentials
+    SSH-->>Gateway: Session established
+    Gateway-->>CLI: Proxied SSH session
+```
+
+## Authentication Methods
+
+Infisical PAM supports three authentication methods for SSH:
+
+| Method | Description | Use Case |
+|--------|-------------|----------|
+| **Password** | Traditional username/password authentication | Legacy systems, simple setups |
+| **SSH Key** | Public key authentication with a private key | Standard secure access |
+| **Certificate** | SSH certificates signed by a CA | Zero-trust, ephemeral credentials |
+
+### Certificate-Based Authentication
+
+Certificate-based authentication is the most secure option. Instead of storing static credentials, Infisical generates short-lived SSH certificates on-demand for each session.
+
+**Benefits:**
+- **No static credentials**: Certificates are generated per-session and expire automatically
+- **Centralized trust**: Servers trust the Infisical CA, not individual keys
+- **Simplified key management**: No need to distribute or rotate SSH keys
+- **Audit trail**: Each certificate is tied to a specific user and session
+
+```mermaid
+sequenceDiagram
+    participant CLI as Infisical CLI
+    participant Gateway
+    participant Infisical
+    participant SSH as SSH Server
+
+    CLI->>Infisical: Request session (authenticate)
+    Infisical-->>CLI: Session ID + Gateway connection details
+    CLI->>Gateway: Connect via secure tunnel
+    Gateway->>Infisical: Fetch credentials for session
+    Infisical->>Infisical: Generate key pair + sign certificate
+    Infisical-->>Gateway: Return private key + certificate
+    Gateway->>SSH: Connect with certificate
+    SSH->>SSH: Verify certificate against CA
+    SSH-->>Gateway: Session established
+    Gateway-->>CLI: Proxied SSH session
+```
+
+## Prerequisites
+
+Before configuring SSH access in Infisical PAM, ensure you have:
+
+1. **An Infisical Gateway** deployed in the same network as your SSH servers
```

Comment:
not necessarily same network.. it just needs to be able to reach the SSH servers